src/app/[lang]/app/search-pro/page.tsx

```diff
@@ -85,10 +85,10 @@ export default function SearchProPage() {
                 <div className='skeleton h-5 w-[250px] rounded' />
               ) : (
                 <span className='text-base font-medium text-dark lg:text-xl'>
-                  We found{' '}
-                  <span className='text-primary'>
-                    {professionalList?.length} pro&apos;s for you
-                  </span>
+                  {intl.formatMessage(
```

Comment:
add condition to show this line only if professional.length > 0

Reply:
We will use different approach with count and remove one of the titles